OpenCDMi/OCDM.cpp

```diff
@@ -111,26 +111,33 @@ namespace Plugin {
         // change to "register" the sink for these events !!! So do it ahead of instantiation.
         _service->Register(&_notification);
 
-        _opencdmi = _service->Root<Exchange::IContentDecryption>(_connectionId, Core::infinite, _T("OCDMImplementation"));
-
-        if (_opencdmi == nullptr) {
-            message = _T("OCDM could not be instantiated.");
+        Plugin::Config::RootConfig rootConfig(service);
+        const uint32_t permission = (S_IRUSR | S_IWUSR | S_IXUSR | S_IRGRP | S_IWGRP | S_IXGRP);
```

Comment:
Do we really need the S_IXUSR and S_IXGRP in this path ?

Reply:
And lets use the OS agnostic flags for this, see: https://github.com/rdkcentral/Thunder/blob/master/Source/core/FileSystem.h#L99-L111